brooklyn-tosca-transformer/src/main/java/io/cloudsoft/tosca/a4c/brooklyn/ToscaPlanToSpecTransformer.java

```diff
@@ -113,6 +114,24 @@ public boolean accepts(String planType) {
         }
     }
 
+
+    public EntitySpec<? extends Application> createApplicationSpec(Path plan) throws PlanNotRecognizedException {
+        assertAvailable();
+        try {
+            Alien4CloudToscaPlatform.grantAdminAuth();
+            return createApplicationSpec(platform.parse(plan));
+        } catch (Exception e) {
+            if (e instanceof PlanNotRecognizedException) {
+                if (log.isTraceEnabled())
```

Comment:
I don't know the original intention. `PlanNotRecognizedException` is a special exception that can be thrown and is treated as a notice rather than a problem by plan to spec transformers. Perhaps it's best to change the `log.debug` statement to `log.trace` then.